lib/oga/xml/document.rb

```diff
@@ -57,6 +57,12 @@ def root_node
         self
       end
 
+      # Returns the top level element in the document.
+
```

Comment:
@scotchi A lack of something offered by other libraries is not a bug, it's simply that: a lack of a feature. Having said that, let's move that to a separate pull request as you suggested.